packages/mdc-auto-init/package.json

```diff
@@ -7,5 +7,8 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/material-components/material-components-web.git"
+  },
+  "dependencies": {
+    "@material/base": "^0.41.0"
```

Comment:
I'm probably missing something obvious but what does this need base for now?

Reply:
Added base imports to index.ts 😄